### PR TITLE
Added SunOS (Solaris, illumos, etc) service management 

### PR DESCRIPTION
**Pull request by szinck**
This uses svcadm and svcs commands to manipulate Solaris services.

### CODE REVIEW

**Comment by rrtj3** [comment]
Nice! I'll give this some testing on solaris 10 tonight.

File: library/service [hunk 1]
```diff
@@ -701,6 +701,128 @@ class NetBsdService(Service):
...
+        # status is one of: online, offline, degraded, disabled, maintenance, uninitialized
+        # see man svcs(1)
+        return status
+
+    def service_enable(self):
+        # Get current service enablement status
+        rc, stdout, stderr = self.execute_command("%s -l %s" % (self.svcs_cmd, self.name))
+
+        if rc != 0:
+            if err:
```
Comment (rrtj3) [inline library/service]: 'err' should be stderr.

**Comment by rrtj3** [comment]
Gave this a run through and it works great, except for the one typo. With that fixed it worked well for my tests. Happy to see this pull request.

**Comment by szinck** [comment]
Thanks for testing this! I really appreciate it.  I'll add a fix for that typo later today.

**Comment by szinck** [comment]
OK, I think this is good to go now

**Comment by mpdehaan** [comment]
merged, thanks!

**Pull request merged**